L1Trigger/L1THGCal/test/testNtuplyzer_cfg.py

```diff
@@ -0,0 +1,103 @@
+process.load('Configuration.StandardSequences.SimL1Emulator_cff')
+process.load('Configuration.StandardSequences.DigiToRaw_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('L1Trigger.L1THGCal.hgcalTriggerPrimitives_cff')
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(-1)
+    )
+
+# Input source
+process.source = cms.Source("PoolSource",
+    secondaryFileNames = cms.untracked.vstring(),
+    fileNames = cms.untracked.vstring(
+        'file:./crab/SinglePhoton/l1tpg_gsdr.root'
+        )
+)
+
+# Additional output definition
+process.TFileService = cms.Service(
+    "TFileService",
+    fileName = cms.string("ntuple.root")
+    )
+
+# Path and EndPath definitions
+process.digitisation_step = cms.Path(process.pdigi_valid)
+process.L1simulation_step = cms.Path(process.SimL1Emulator)
+
+# Remove best choice selection
+process.hgcalTriggerPrimitiveDigiProducer.FECodec.NData = cms.uint32(999)
+process.hgcalTriggerPrimitiveDigiProducer.FECodec.DataLength = cms.uint32(8)
+process.hgcalTriggerPrimitiveDigiProducer.FECodec.triggerCellTruncationBits = cms.uint32(7)
+
+process.hgcalTriggerPrimitiveDigiProducer.BEConfiguration.algorithms[0].calib_parameters.cellLSB = cms.double(
+        process.hgcalTriggerPrimitiveDigiProducer.FECodec.linLSB.value() * 
+        2 ** process.hgcalTriggerPrimitiveDigiProducer.FECodec.triggerCellTruncationBits.value() 
+)
+
+
+trgCells_algo_all =  cms.PSet( AlgorithmName = cms.string('SingleCellClusterAlgoBestChoice'),
+                              FECodec = process.hgcalTriggerPrimitiveDigiProducer.FECodec,
+                              HGCalEESensitive_tag = cms.string('HGCalEESensitive'),
+                              HGCalHESiliconSensitive_tag = cms.string('HGCalHESiliconSensitive'),
+                           
+                              calib_parameters = process.hgcalTriggerPrimitiveDigiProducer.BEConfiguration.algorithms[0].calib_parameters
+                              )
+cluster_algo_all =  cms.PSet( AlgorithmName = cms.string('HGCClusterAlgoBestChoice'),
+                              FECodec = process.hgcalTriggerPrimitiveDigiProducer.FECodec,
+                              HGCalEESensitive_tag = cms.string('HGCalEESensitive'),
+                              HGCalHESiliconSensitive_tag = cms.string('HGCalHESiliconSensitive'),                           
+                              calib_parameters = process.hgcalTriggerPrimitiveDigiProducer.BEConfiguration.algorithms[0].calib_parameters,
+                              seeding_threshold = process.hgcalTriggerPrimitiveDigiProducer.BEConfiguration.algorithms[0].C2d_parameters.seeding_threshold,
+                              clustering_threshold = process.hgcalTriggerPrimitiveDigiProducer.BEConfiguration.algorithms[0].C2d_parameters.clustering_threshold,
```

Comment:
C2d parameters should be grouped together (as well as C3d), as done for calibration parameters